Automattic/woocommerce-payments
# Add E2E Merchant / Order / Full Refund

Fixes https://github.com/Automattic/woocommerce-payments/issues/1485

#### Changes proposed in this Pull Request

This PR adds a new test covering the `Merchant / Order / Full Refund` [critical flow](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows).

Additionally this PR adds the following:

* A new StoreOwnerFlow `goToOrder ( orderId )` that loads the order details view
* A new helper method to wait for the UI placeholder to finish loading and the content to display

#### Testing instructions

* Make sure the new test passes by running: `npm run test:e2e tests/e2e/specs/merchant/order-full-refund.spec.js`
  * Alternatively, run it in non-headless mode:`npm run test:e2e-dev tests/e2e/specs/merchant/order-full-refund.spec.js`
* Make sure all tests pass `npm run test:e2e`
-------------------
